From f43ecb1ca5f309f42ebc38e2209fab9332a5193b Mon Sep 17 00:00:00 2001
From: Alexandre Torgue <alexandre.torgue@foss.st.com>
Date: Mon, 22 Oct 2018 15:31:06 +0200
Subject: [PATCH 008/529] ARM: configs: add multi_v7_defconfig addons fragment
 for stm32mp1

Add addons fragment to add configs used by stm32mp1.

Change-Id: I617e1b0d2381a62e015e642a4a96f76a3dd73875
Signed-off-by: Alexandre Torgue <alexandre.torgue@foss.st.com>
Signed-off-by: Amelie Delaunay <amelie.delaunay@foss.st.com>
---
 .../configs/fragment-02-multiv7_addons.config | 38 +++++++++++++++++++
 1 file changed, 38 insertions(+)
 create mode 100644 arch/arm/configs/fragment-02-multiv7_addons.config

--- /dev/null
+++ b/arch/arm/configs/fragment-02-multiv7_addons.config
@@ -0,0 +1,38 @@
+#  ALPHABETICALLY SORTED: $LC_ALL=C sort fragment-02-multiv7_addons.config -o fragment-02-multiv7_addons.config
+CONFIG_AEABI=y
+CONFIG_CHR_DEV_SG=y
+CONFIG_CRC_ITU_T=m
+CONFIG_DEBUG_INFO=y
+CONFIG_DEBUG_INFO_DWARF_TOOLCHAIN_DEFAULT=y
+CONFIG_DEBUG_PREEMPT=y
+CONFIG_DEBUG_SECTION_MISMATCH=y
+CONFIG_DNS_RESOLVER=y
+CONFIG_DYNAMIC_DEBUG=y
+CONFIG_FORCE_MAX_ZONEORDER=12
+CONFIG_FW_LOADER_USER_HELPER_FALLBACK=y
+CONFIG_GDB_SCRIPTS=y
+CONFIG_HIGHMEM=y
+CONFIG_IKCONFIG=y
+CONFIG_IKCONFIG_PROC=y
+CONFIG_KEYS=y
+CONFIG_LOG_BUF_SHIFT=16
+CONFIG_MCPM=y
+CONFIG_MEMCG=y
+CONFIG_NAMESPACES=y
+CONFIG_NR_CPUS=2
+CONFIG_OVERLAY_FS=y
+CONFIG_POSIX_MQUEUE=y
+CONFIG_POWER_RESET=y
+CONFIG_POWER_RESET_SYSCON=y
+CONFIG_PREEMPT=y
+CONFIG_PREEMPT_COUNT=y
+CONFIG_PROFILING=y
+CONFIG_SENSORS_IIO_HWMON=y
+CONFIG_SERIAL_NONSTANDARD=y
+CONFIG_SRAM=y
+CONFIG_THERMAL=y
+CONFIG_TMPFS=y
+CONFIG_USELIB=y
+CONFIG_VFP=y
+CONFIG_ZBOOT_ROM_BSS=0x0
+CONFIG_ZBOOT_ROM_TEXT=0x0
